Tests/SubprocessTests/SubprocessTests+Unix.swift

```diff
@@ -668,14 +668,11 @@ extension SubprocessUnixTests {
         var platformOptions = PlatformOptions()
         platformOptions.supplementaryGroups = Array(expectedGroups)
         let idResult = try await Subprocess.run(
-            .name("swift"),
+            .path("/usr/bin/swift"),
```

Comment:
Does this really still hang without this change, considering the brand-new I/O backend? Could we revert this change? (it's useful for platforms where swift is not located in /usr/bin)

Reply:
It's actually due to https://github.com/swiftlang/swift-subprocess/issues/105 which I addressed. I reverted it back to `.name()`.

Reply:
Oh, nice!